packages/interface-ipfs-core/src/object/links.js

```diff
@@ -60,7 +60,7 @@ module.exports = (common, options) => {
       const node1bCid = await ipfs.object.put(node1b)
 
       const links = await ipfs.object.links(node1bCid)
-      expect(node1b.Links[0]).to.eql({
+      expect(node1b.Links[0]).to.containSubset({
```

Comment:
This has broken because [this PR](https://github.com/ipld/js-ipld-dag-pb/pull/184) changed `DagPBNode.Links` to make it [return DAGLinks](https://github.com/ipld/js-ipld-dag-pb/commit/0f11d5c1f42df0f6f9e6ec850f1c2acf0d52adf0#diff-a144e7bd8c520628cd0cdc5279f2f900R37) where it used to [return plain objects](https://github.com/ipld/js-ipld-dag-pb/commit/0f11d5c1f42df0f6f9e6ec850f1c2acf0d52adf0#diff-a144e7bd8c520628cd0cdc5279f2f900L108-L116), making it a breaking change.  Thankfully it was released as `0.x.0` so hasn't been pulled in by people ahead of merging this PR 😥 

```suggestion
      expect(node1b.Links[0]).to.deep.equal(links[0])
```